orchestrator/test_runner/src/eip_712.rs

```diff
@@ -0,0 +1,112 @@
+use clarity::Address as EthAddress;
+use deep_space::{Address as CosmosAddress, Coin, Contact, CosmosPrivateKey};
+use gravity_proto::gravity::query_client::QueryClient as GravityQueryClient;
+use num256::Uint256;
+use reqwest::Client;
+use tonic::transport::Channel;
+use web30::client::Web3;
+
+use crate::{utils::ValidatorKeys, OPERATION_TIMEOUT, STAKING_TOKEN};
+
+pub const TX_POST_PORT: &str = "1317";
+pub const TX_POST_ENDPOINT: &str = "/cosmos/tx/v1beta1/txs";
+pub const SUCCESSFUL_EIP712_MSG_SEND: &str = "10,209,2,10,140,1,10,28,47,99,111,115,109,111,115,46,98,97,110,107,46,118,49,98,101,116,97,49,46,77,115,103,83,101,110,100,18,108,10,46,103,114,97,118,105,116,121,49,104,97,110,113,115,115,54,106,115,113,54,54,116,102,121,106,122,53,54,119,122,52,52,122,48,101,106,116,121,118,48,55,50,52,104,51,50,99,18,46,103,114,97,118,105,116,121,49,57,48,101,53,108,48,107,121,100,100,100,52,108,106,108,50,54,50,102,120,101,102,110,110,114,102,103,50,50,104,50,114,116,115,50,118,52,107,26,10,10,5,115,116,97,107,101,18,1,49,18,23,84,101,115,116,32,69,73,80,45,55,49,50,32,83,105,103,110,97,116,117,114,101,115,250,63,165,1,10,42,47,101,116,104,101,114,109,105,110,116,46,116,121,112,101,115,46,118,49,46,69,120,116,101,110,115,105,111,110,79,112,116,105,111,110,115,87,101,98,51,84,120,18,119,8,191,132,61,18,46,103,114,97,118,105,116,121,49,104,97,110,113,115,115,54,106,115,113,54,54,116,102,121,106,122,53,54,119,122,52,52,122,48,101,106,116,121,118,48,55,50,52,104,51,50,99,26,65,120,230,150,225,9,81,29,44,180,172,236,26,57,239,59,82,51,251,187,220,116,112,145,193,251,194,160,245,192,203,165,177,60,110,18,182,62,81,38,130,26,111,165,87,175,230,37,129,148,216,35,76,148,29,40,98,44,66,212,86,193,44,180,16,27,18,107,10,87,10,79,10,40,47,101,116,104,101,114,109,105,110,116,46,99,114,121,112,116,111,46,118,49,46,101,116,104,115,101,99,112,50,53,54,107,49,46,80,117,98,75,101,121,18,35,10,33,2,96,22,221,190,65,131,210,254,152,203,219,14,220,50,31,66,250,207,36,246,56,139,39,117,192,120,199,253,189,89,254,2,18,4,10,2,8,127,18,16,10,10,10,5,115,116,97,107,101,18,1,49,16,128,137,122,26,0";
```

Comment:
could we add an unsuccessful use case, like one where the signature is off by one byte? I know you encountered a lot of those getting this working, but we should enshrine one to test with just to project against a situation in the future where the signature might not actually be checked.